### PR TITLE
Public dataset fixes

### PR DESCRIPTION
**Pull request by xavijam**
- [x] One click or edit in CartoDB button appearance.
- [x] No table hover.
- [x] No table double click.
- [x] Limit maps related with public dataset to 3.
- [x] Fixed "no rows matched" with public map view problem in export view.
- [x] User breadcrumb style.
- [x] Remove public table shadow.

cc @saleiva

### CODE REVIEW

File: app/views/admin/visualizations/public_dataset.html.erb [hunk 1]
```diff
@@ -234,7 +234,7 @@
 
     <% if @total_visualizations.count > 0 %>
       <ul class="MapsList PublicMap-mapsList">
-        <% @total_visualizations.each do |vis| %>
+        <% @total_visualizations.take(3).each do |vis| %>
```
Comment (xavijam) [inline app/views/admin/visualizations/public_dataset.html.erb]: @javisantana ```take``` function is ok right? Just for limiting the number of maps in the "Maps created with this dataset", if not all maps will appear.

**Comment by saleiva** [comment]
:es:

File: app/views/admin/visualizations/public_dataset.html.erb (hunk 1 above)
Comment (javisantana) [inline app/views/admin/visualizations/public_dataset.html.erb]: no idea :D

Comment (xavijam) [inline app/views/admin/visualizations/public_dataset.html.erb]: Checked we use it in is other places:
https://github.com/CartoDB/cartodb/blob/master/lib/cartodb/stats/api_calls.rb#L130

**Pull request merged**